Firestore/core/src/firebase/firestore/immutable/sorted_map_iterator.h

```diff
@@ -119,7 +119,9 @@ class SortedMapIterator {
   pointer get() const {
     switch (tag_) {
       case Tag::Array:
-        return array_iter_;
+        // In the Microsoft STL, std::array::iterator is not a bare pointer so
```

Comment:
Optional: consider rephrasing in a slightly more generic manner, e.g. `std::array::iterator is not guaranteed to be a bare pointer` (with or without the `so this takes the address...` part which I think can be omitted because it just restates the next line of code).

Reply:
Rephrased.